plugins/catalog-backend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@backstage/plugin-catalog-backend",
-  "version": "0.2.2",
+  "version": "0.3.0",
```

Comment:
These version bumps shouldn't be included, they happen automatically based on the changesets in the release PR

Reply:
I've reverted `catalog-backend` version to 0.2.2 and reverted it's dependency versions to 0.2.2 in `example-backstage` and `catalog-import` packages